### ISSUE TITLE
Mattermost web crashes when removing the "I’ve deleted the reminder" post

### ISSUE THREAD

**Issue by YuKitAs** [scottleedavis/mattermost-plugin-remind#132]
**Reproduce**:
1. Create a new reminder task
2. In the reply "I will remind ***", click the `Delete` button below
3. In the reply "I’ve deleted the reminder ***", click the `x` button at the right corner (in order to remove the post)
4. Mattermost will jump to a blank page (`/messages/@remindbot`)

**Expected**:
The post will be removed normally.

**Mattermost version**: 5.12.1 (Webversion on Chrome and Firefox)

**Comment by scottleedavis** [comment]
verified on 5.12.1.  Discussion with mattermost team possible causes.

**Comment by scottleedavis** [comment]
Still no resolution, however I am tempted to create a simple plugin to replicate the issue to then present again to the team.  The mattermost-plugin-demo does not have this issue and also uses ephemeral posts.

**Comment by YuKitAs** [comment]
Hi, thanks for following up! My mattermost was upgraded to v5.13.1 but the issue still exists. 
When I looked at the response for the blank page, the logs showed 
```
{"level":"ERROR","message":"msg: Uncaught TypeError: Cannot read property 'id' of undefined row: 1 col: 402495 stack: TypeError: Cannot read property 'id' of undefined url: https://mattermost.gameforge.com/static/main.b9238839ce0ad2b12267.js"}"
```

**Comment by scottleedavis** [comment]
@YuKitAs turns out it was a bug in the code.  Fixed in v0.4.3
https://github.com/scottleedavis/mattermost-plugin-remind/releases/tag/0.4.3
